task_manager/tests/conftest.py

```diff
@@ -1,7 +1,24 @@
+from typing import Generator
+
 import pytest
 from django.test import Client
+from freezegun import freeze_time
+from task_manager.users.models import ServiceUser
 
 
 @pytest.fixture
 def client() -> Client:
     return Client()
+
+
+@pytest.fixture
+def service_user() -> ServiceUser:
+    return ServiceUser.objects.create(
+        first_name="ilon", last_name="mask", username="ilon-mask", password="%%%%%"
+    )
+
+
+@pytest.fixture(autouse=True)
```

Comment:
in the current implementation it's going to be called for each test, wdyt does it make sense to call it once for the test execution session using [session-scope](https://docs.pytest.org/en/7.1.x/how-to/fixtures.html#fixture-scopes) fixture?

Reply:
done